### ISSUE TITLE
Undefined symbols for architecture x86_64:   "_OBJC_CLASS_$_UIPointerShape", referenced from:       objc-class-ref in GoogleMaps

### ISSUE THREAD

**Issue by mitjnextt** [react-native-community/react-native-maps#3526]
I am using Google Maps in iOS,
after pod installing (pod 'GoogleMaps') My XCode build is failing with following

```
Undefined symbols for architecture x86_64:
  "_OBJC_CLASS_$_UIPointerShape", referenced from:
      objc-class-ref in GoogleMaps
  "_OBJC_CLASS_$_UIPointerStyle", referenced from:
      objc-class-ref in GoogleMaps
ld: symbol(s) not found for architecture x86_64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

My React native version is...
"react": "16.11.0",
"react-native": "0.62.2",
"react-native-maps": "^0.27.1",

**Comment by mitjnextt** [comment]
Resolved it by downgrading GoogleMaps version to...

pod 'GoogleMaps', '2.5.0'

**Comment by mitjnextt** [comment]
@christopher-18 Have you added the AirGoogleMaps folder to your XCode?

**Comment by mitjnextt** [comment]
![image](https://user-images.githubusercontent.com/67790880/91403317-f3054480-e85e-11ea-9687-c5dbbeb85f6c.png)

Also while copy AirGoogleMaps,
Select Create Groups & Copy items if needed
Do not select create folder reference

**Comment by christopher-18** [comment]
@mitjnextt Thanks for the reply. I got it working 2 days back by doing the same along with adding the airmaps path to the header search paths and setting USE_GOOGLE_MAPS = 1 in preprocessor macros.

**Comment by kreshikhin** [comment]
UIPointerShape is available from ios 13.4+ (https://developer.apple.com/documentation/uikit/uipointershape).
As far as I figured out GoogleMaps 3.10.0 requires this stuff and fails on old projects (old SDK and target < ios 13.4). But 3.9.0 compiles well without this issue.

Maybe this info be helpful for somebody.
